### PR TITLE
init

### PR DESCRIPTION
**Pull request by ZidiMedAmine**

### CODE REVIEW

**Comment by ZidiMedAmine** [comment]
i'm happy that i contributed in this project

File: hello.txt [hunk 1]
```diff
@@ -1 +1 @@
-2211 new new hello
\ No newline at end of file
+2211 new new hellosdsds
```
Comment (khalilFendri) [inline hello.txt:1]: what is this dog water!

**Comment by khalilFendri** [comment]
dfjgslfjg

File: hello.txt (hunk 1 above)
Comment (khalilFendri) [inline hello.txt:1]: lkdjf

Comment (khalilFendri) [reply hello.txt:1]: sjdfhqsd

**Comment by khalilFendri** [comment]
lllllllllllllllllllllllllllllllllllllll

**Comment by khalilFendri** [comment]
@ZidiMedAmine hi

**Comment by jihencherni123** [comment]
hello @khalilFendri

**Comment by khalilFendri** [comment]
@jihencherni123 tag back

**Comment by khalilFendri** [comment]
kafka yo

**Comment by khalilFendri** [comment]
test

**Comment by khalilFendri** [comment]
ljsdf lkjmds

**Comment by khalilFendri** [comment]
qsd fk

**Comment by khalilFendri** [comment]
lqjshd

**Comment by khalilFendri** [comment]
qsd as

**Comment by khalilFendri** [comment]
dfsqd

**Comment by khalilFendri** [comment]
sdfsdf

**Comment by khalilFendri** [comment]
hlfhf

**Comment by khalilFendri** [comment]
sdqsd

**Comment by khalilFendri** [comment]
dsqfqsd

**Comment by khalilFendri** [comment]
fqsdf

**Comment by khalilFendri** [comment]
sdff

**Comment by khalilFendri** [comment]
sqdfqsdf

**Comment by khalilFendri** [comment]
qqqqqqqqqqqqqqqq

**Comment by khalilFendri** [comment]
sdmfksd

**Comment by khalilFendri** [comment]
hdsfjhskdf

**Comment by khalilFendri** [comment]
Fg

**Comment by khalilFendri** [comment]
mùkshdfsd

**Comment by khalilFendri** [comment]
ùlqsdhf

**Review by khalilFendri — APPROVED** [review]

**Pull request merged**